src/server.mjs

```diff
@@ -382,6 +382,49 @@ api.post("/getChunkedData", (req, res, next) => {
   });
 });
 
+
+/*
+graph = {
+  node: [
+    {
+      sequence: "AGCT"
+      id: "1"
+    ,
+    {
+      sequence: "AGCTAG"
+      id: "2"
+    }
+  ]
+  edge: []
+  path: []
+}
+removing sequence would result in
+graph = {
+  node: [
+    {
+      id: "1"
+    ,
+    {
+      id: "2"
+    }
+  ]
+  edge: []
+  path: []
+}
+*/
+
+// read a graph object and remove "sequence" fields in place
+function removeNodeSequencesInPlace(graph){
+  console.log("graph:", graph)
+  if (!graph.node){
+    return;
+  }
+  graph.node.forEach(function(node) {
+    console.log("node sequence to delete:", node)
```

Comment:
We probably should cut this; people are going to remove sequence because there's a lot of it, and we don't want a lot of sequence going to the logs.
```suggestion
```